templates/setup.py.jj2

```diff
@@ -84,12 +85,20 @@ CLASSIFIERS = [
     'Topic :: Software Development :: Libraries',
     'Programming Language :: Python',
     'Intended Audience :: Developers',
+    {% if not min_ver_python[0] == 3 %}
     'Programming Language :: Python :: 2.6',
     'Programming Language :: Python :: 2.7',
+    {% endif %}
+
+    {% if min_ver_python[0] == 3 %}
+    'Programming Language :: Python :: 3 :: Only',
+    {% endif %}
+
     'Programming Language :: Python :: 3.3',
```

Comment:
It would be good if we add something like:-
```
{% if min_ver_python < [3,4] %}
    'Programming Language :: Python :: 3.3',
{% endif %}
```
This would allow the accessibility of only displaying the classifiers which are required. You could do something similar to this to all the python version.